GDI spec - configuration module (#45)
signalfx/splunk-otel-java-lambda: README.md

This Splunk distribution comes with the following defaults:

> - `tracecontext,baggage` context propagation

Reviewer: This particular string might not mean anything to end users. I think we should say something like "W3C specified Trace Context and Baggage propagation".

Author: Good point.

- OpenTelemetry Protocol (`otlp`) traces exporter

> - no metrics exporter

Reviewer: ```suggestion
- No metrics exporter
```

   For more information about setting environment variables in the AWS console,
   see [Using AWS Lambda environment variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html)
   on the AWS website.

> 5. By default, the Splunk Lambda wrapper uses `tracecontext,baggage` context propagation. 

Reviewer: same comment here about this not being meaningful to end-users.

Author: +1

    
   If you want to change this, set the `OTEL_PROPAGATORS` environment variable in your
   Lambda function code. For more information about available context
   propagators, see the [Propagator settings](https://github.com/open-telemetry/opentelemetry-java/tree/v1.1.0/sdk-extensions/autoconfigure#customizing-the-opentelemetry-sdk)
   for the OpenTelemetry Java.

> 6. By default, the Splunk Lambda wrapper uses a `otlp` exporter to send traces to Splunk APM. 

Reviewer: ```suggestion
6. By default, the Splunk Lambda wrapper uses an OpenTelemetry Protocol (`otlp`) exporter to send traces to Splunk APM. 
```

> 6. By default, the Splunk Lambda wrapper uses an OpenTelemetry Protocol (`otlp`) exporter to send traces to Splunk APM. 

Reviewer 2: ```suggestion
6. By default, the Splunk Lambda wrapper uses the OpenTelemetry Protocol (`otlp`) exporter to send traces to Splunk APM. 
```
